community/contribution.md

```diff
@@ -15,28 +15,34 @@ _This document explains how Axelerant defines contributions with respect to KPIs
 
 An open source community contribution is defined as any piece of documentation (code or otherwise) or any offline activity that is documented and available in the public space for other community members to use.
 
+### Axelerant’s intent
+
+While we understand the above is an open source contribution **Axelerant’s intent is to encourage more meaningful contributions for the community & for the organisation.**
+
+## Important Notes before you continue reading:
```

Comment:
I guess then this could be moved to the very top of the document. It doesn't make sense in its current position. However, the last point (and maybe others) don't make a lot of sense in this section. The only points with real substance here are points 3 and 4. The rest are more of a clarification and "where to look for support".

So, my suggestion is to move points 3 and 4 to a better section about contributions and KPI. The other points can remain in this block and may be moved to the top.